scripts/firmware.py

```diff
@@ -105,12 +100,57 @@ class FirmwareFile(NamedTuple):
 
 
 class Utils:
+    CONFIG_FILE: ClassVar[str] = "firmware.cfg"
+
+    # Default Directories
+    DEFAULT_DIRECTORIES: ClassVar[Dict[str, str]] = {
+        "KLIPPY_LOG": os.path.expanduser("~/printer_data/logs/klippy.log"),
+        "KATAPULT_DIR": os.path.expanduser("~/katapult"),
+        "KLIPPER": os.path.expanduser("~/klipper"),
+        "KLIPPY_ENV": os.path.expanduser("~/klippy_env"),
+    }
+
+    @classmethod
+    def load_config(cls) -> Dict[str, str]:
+        """
+        Load variables from the configuration file if it exists.
+        Fall back to default values if the file is not found.
+
+        Returns:
+            Dict[str, str]: A dictionary containing the configuration variables.
+        """
+        config_variables: Dict[str, str] = dict(cls.DEFAULT_DIRECTORIES)
+        logging.debug(f"Loading Config Variables: {config_variables}")
```

Comment:
This outputs the default config variables, I worry that can confuse people.
Shouldn't we just have one output with what it loaded?